o2a/converter/oozie_converter.py

```diff
@@ -135,6 +149,20 @@ def create_dag_file(self, workflow: Workflow):
         black.format_file_in_place(
             Path(file_name), mode=black.FileMode(line_length=110), fast=False, write_back=black.WriteBack.YES
         )
+        fix_file(
```

Comment:
Can you create a method to separate the roles of this function?

Reply:
Sure. Good idea!

Reply:
Done